content/post/057-ipfs-desktop-0-9.md

```diff
@@ -0,0 +1,65 @@
+
+### Browse the wild...
+
+With the recent update to the Web UI, we are now capable of navigating through the whole IPFS system. Previously, you could only access your 'Files' (the contents of the [Mutable File System](https://docs.ipfs.io/guides/concepts/mfs/) for those who enjoy technical terms). Now, you can navigate not only to any `/ipfs/` or `/ipns/` path, but also any content [pinned](https://docs.ipfs.io/guides/concepts/pinning/) to your IPFS node!
+
+![Browing /ipns/ipns.io](/057-ipfs-desktop-0-9/browse-wild.png)
+
+### Use npm on ipfs
+
+In addition, we are introducing a new experiments section on the 'Settings' page which will allow you to enable new experimental features as they come in. To celebrate this new section, we are adding [npm on ipfs](https://github.com/ipfs-shipyard/npm-on-ipfs) to it! This is a project that allows you to use the `ipfs-npm` command to install your Node.js modules through the IPFS network!
+
+![Experiments section](/057-ipfs-desktop-0-9/npm-ipfs.png)
+
+**Please note** that this feature is intended for those who have Node.js installed on their system. Moreover, the activation will only work if your setup does not require additional permissions for executing `npm install -g`.
+
+### Add `ipfs` command line tools to your system
+
+Adding `ipfs` command line tools to your system was actually introduced in 0.8.0, but it's always worth noting that you are just a click away of having the `ipfs` command available from your command line! This way, you don't have to worry about configuring your `PATH` variable or doing other confusing stuff to start using all IPFS capabilities. Just go to Settings and enable "IPFS command line tools"!
+
+![IPFS command line tools](/057-ipfs-desktop-0-9/ipfs-cmd.png)
+
+### And more...
+
+Other notable features from previous releases include:
+
+- **Handle `ipfs://`, `ipns://` and `dweb:` links**. If you have IPFS Desktop installed and you click on [ipns://ipfs.io](ipns://ipfs.io), that request will go through be redirected to your own gateway via the app, or to the public gateway if yours is not online.
+- **Easily add files to IPFS**. Just drag and drop them to the application icon, either on the menubar for macOS users, of the shortcut icon for Windows users.
+- **Add screenshots to IPFS**. You can click on 'Take Screenshot' on the app's menu or enable the global shortcut on "Settings". After taking a screenshot, a shareable link will be copied to your clipboard!
+- **Download any CID content**. If you would like to download the contents of some CID or IPFS/IPNS path, you can copy it, go to the dropdown menu and select 'Download Hash'. Then, just pick wherever you want to write those files to! This is also available through a global shortcut.
```

Comment:
Can you note what the global shortcut is for each OS?

Reply:
Hmmm... I wonder if it is really necessary to have on the blog post. I am saying this because these are features that are required to be manually enabled. Thus, the user will always see the shortcuts on settings